docs/readthedocs_theme/base.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   {% block htmltitle %}
   <title>{{ page_title }}</title>
+  {% if page_description %}<meta name="description" content="{{ page_description }}">{% endif %}
```

Comment:
Ideally we would have a title for the frontpage and a different title for subpages which have their one title.